extensions/compose/src/compose-extension.ts

```diff
@@ -135,6 +135,21 @@ export class ComposeExtension {
       this.statusBarItem.show();
       this.extensionContext.subscriptions.push(this.statusBarItem);
     }
+    const description = `Compose is a tool for defining and running multi-container Docker applications. 
+    With Compose, you use a YAML file to configure your application's services. Then, with a single command, 
+    you create and start all the services from your configuration.
+    Compose works in all environments; production, staging, development, testing, as well as CI workflows. 
+    It also provides commands for managing the whole lifecycle of your application.`;
+
+    extensionApi.cli.createCliTool({
```

Comment:
Should we add a note like.. "TODO: Remove before merging?" so we don't merge this in by mistake?

Reply:
moved to separate PR https://github.com/containers/podman-desktop/pull/4388